packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -463,6 +463,13 @@ export class Property extends ElementPatchOperation<Opaque> {
 
   protected update(dom: DOMHelper, element: Element, value: Opaque) {
     let { name } = this;
+
+    // Needed for compatibility with Microsoft browsers
+    // Copied from https://github.com/tildeio/htmlbars/blob/ec541e79dbf0b03471070a0de3ac810f8fc924ed/packages/dom-helper/lib/main.js#L309-L319
+    if (value === null && (name === 'value' || name === 'type' || name === 'src')) {
```

Comment:
This makes me somewhat sad.  Can you create a subclass of `Property` for this (and use it in `addAttribute` as you have)?

```js
class EmptyStringResetingProperty extend Property {
  protected update(dom: DOMHelper, element: Element, value: Opaque) {
    if (value === null) {
      value = '';
    }
    
    super.update(dom, value);
  }
}
```